### DISCUSSION TITLE
Error with CSP and livewireScripts

### DISCUSSION THREAD

**Discussion by olezhkafp** [livewire/livewire#3894] [7. Bugs (archived)]
Hi. I already read all closed issues related to this one, but looks like this is new error happned there.
I use https://github.com/spatie/laravel-csp package.
I've installed livewire and include it in blade using this code:

`<?php $nonce = ["nonce" => csp_nonce()] ?>`
`@livewireScripts($nonce)`

or in this way:

`@livewireScripts(['nonce' => csp_nonce()])`

if I just render `$nonce` variable - I see it in HTML:
`<div>{{print_r($nonce, true)}}</div>`

but, livewire scripts does not render it - there is just empty strings

![U85RN9IURLirUBVHXDO6Xw](https://user-images.githubusercontent.com/11082011/135074730-604c887a-a377-402f-89b0-653ae05d16b1.png)

PHP: 7.4.11
livewire verison: v2.6.7
laravel: v8.61.0
OS: windows 10

**Comment by joshhanley** [comment]
@olezhkafp so I just did a test `@livewireScripts(['nonce' => 'test'])` and it seems to be outputting correctly

![image](https://user-images.githubusercontent.com/882837/139558949-bb9348ef-02ee-44eb-8100-30d6db146f14.png)

So maybe there is an issue with how `csp_nonce()` is outputting or maybe it's in a different format?

For reference, I tested on Livewire 2.7.2, but I don't think anything should have changed.

Hope this helps!
